### PR TITLE
Feature 2 add middle section mobile

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
### In this project we :
- Add second section ( Work - About ). 
- Add Linters for HTML/CSS.
- Use Mobile-first approach.
- Use best practices for HTML & CSS.
- Update README file.
- Add Mark up to the .html file.
- Add style to the main header.
- Use images and backgrounds to enhance the look of the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,74 +12,235 @@
...
+                <span>Skils</span>
+                <ul class="list">
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>
+        </section>
```
Comment (franclobo) [inline index.html:65]: In the case of Figma template # 4, these skills replace the About section. 👆🏽

File: index.html [hunk 2]
```diff
@@ -12,74 +12,235 @@
...
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-            </ul>
-        </div>
-    </section>
+        <img class="work" src="img/Element-Right.png" alt="">
+    </div>
+
+    <!-- ----SECOND SECTION START -------->
```
Comment (franclobo) [inline index.html:98]: - [ ] Please 🙏🏽 the Work section should be inside the `<section>` tag for the best practices for HTML & CSS.

File: styles.css [hunk 3]
```diff
@@ -154,3 +166,114 @@ main p {
...
+  background: url("/img/portfolio.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: fit-content;
+}
+
+.work-titl {
+  font-size: 40px;
+  font-weight: 500;
```
Comment (franclobo) [inline styles.css:186]: - [ ] Kindly correct the font -weight to be similar to the Figma template. 👇🏽

image

File: index.html [hunk 4]
```diff
@@ -12,74 +12,235 @@
...
+            </a>
+            <a href="#" rel="noopener noreferrer nofollow" target="_blank">
+                <img src="/img/Angelist.png" alt="">
+            </a>
+            <a href="https://twitter.com/Mostafa99738811" rel="noopener noreferrer nofollow" target="_blank">
+                <img src="img/Vectotr.png" alt="twitter">
+            </a>
+            <a href="#" rel="noopener noreferrer nofollow" target="_blank">
+                <img src="img/Vector1.png" alt="">
+            </a>
```
Comment (franclobo) [inline index.html]: - [ ] Kindly, complete the icons there are 5 in the Figma template, and make sure that they show in the browser. 👇🏽 
- [ ] Please 🙏🏽 use the `<ul>` tag like a container of the icons, for the best practices for HTML & CSS.

- Your design

image

- Figma template
image

File: index.html [hunk 5]
```diff
@@ -12,74 +12,235 @@
...
+                </a>
+                <a href="https://twitter.com/Mostafa99738811" rel="noopener noreferrer nofollow" target="_blank">
+                    <img src="img/Vectotr.png" alt="twitter">
+                </a>
+                <a href="#" rel="noopener noreferrer nofollow" target="_blank">
+                    <img src="img/Vector1.png" alt="">
+                </a>
+                <img class="hr" src="img/Vector-4.png" alt="hrl">
+            </div>
+        </article>
```
Comment (franclobo) [inline index.html]: - [ ] This About section is one with the Home page in Figma template # 4. Kindly delete this section to look similar to the Figma template.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @MostafaShahat2021, 

You are doing it well, congratulations!!! 😄 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️
- Correct Github flow 💯 
- Descriptive PR ✔️ ✔️ 
- Professional README file 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please 🙏🏽 review the Figma [template # 4](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) and make your portfolio similar to this template. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hi @franclob
Thank you for your efforts in reviewing my code,
1- Add work section inside section tag.
2- Make sure that icons show in the browser.
3- make sure that my recent work title has the same font-weight as the Figma template.
thank you, have a good day,
Best Regards.
fix issues

File: index.html [hunk 6]
```diff
@@ -12,74 +12,165 @@
...
+            </p>
+            <div class="list-container">
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li class="ruby">Ruby</li>
+                </ul>
+            </div>
+            <button type="button" class="btn">See Project</button>
```
Comment (codecaiine) [inline index.html:136]: - [ ] [OPTIONAL] Good job so far :smile: but please follow the button styling: Active, Hover and Pressed  as defined on Figma design :smile: 
Screenshot from 2022-08-24 15-40-58

**Review by codecaiine — APPROVED** [review]
Hi @MostafaShahat2021 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great project implementation :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
*Hi ***@***.****
*Thank you for your efforts in reviewing my code,*
*1- Add work section inside section tag.*
*2- Make sure that icons show in the browser.*
[image: image.png]

*3- make sure that my recent work title has the same font-weight as the
Figma template.*
[image: image.png]


*thank you, have a good day,*
*Best Regards.*

**Pull request merged**